sentry-core/src/logs.rs

```diff
@@ -0,0 +1,392 @@
+/// Captures a log at the given level, with the given message and attributes.
+/// The attributes are passed as `key = value` arguments before the message, which can be a simple string or a format string with its arguments.
+///
+/// The supported attribute keys are all valid Rust identifiers with up to 8 dots.
+/// Using dots will nest multiple attributes under their common prefix in the UI.
+///
+/// The supported attribute values are simple types, such as string, numbers, and boolean.
+///
+/// See also the [`crate::trace!`], [`crate::debug!`], [`crate::info!`], [`crate::warn!`], [`crate::error!`], and [`crate::fatal!`] macros, which call `log!` with the corresponding level.
```

Comment:
as these will get rendered to rustdoc, using ``[`sentry::trace!`](crate::trace!)`` should probably do the trick of generating a nice looking (and correct) link.